jkube-kit/enricher/generic/src/test/java/org/eclipse/jkube/enricher/generic/IngressEnricherTest.java

```diff
@@ -219,7 +222,7 @@ void getIngressTlsXMLConfig_withNullResourceConfig() {
         List<IngressTlsConfig> ingressTlsConfigs = IngressEnricher.getIngressTlsXMLConfig(null);
 
         // Then
-        assertThat(ingressTlsConfigs).asList().isEmpty();
+        assertThat(ingressTlsConfigs).asInstanceOf(InstanceOfAssertFactories.list(IngressTlsConfig.class)).isEmpty();
```

Comment:
https://github.com/eclipse-jkube/jkube/pull/3321/files#r1719649078

Reply:
yes, agree. I will modify this